File: Algorithm/src/greedy/LeetCode455.java
package greedy;

import java.util.Arrays;

public class LeetCode455 {

    public int findContentChildren(int[] g, int[] s) {

        Arrays.sort(g);
        Arrays.sort(s);

        int j = 0;
        for(int i = 0; i <s.length && j < g.length ; i++){
            if(s[i] >= g[j]){
                j++;
            }
        }
        return j;
    }
}

File: HOT/greedy/LeetCode455.go
package greedy

import "sort"

func findContentChildren(g []int, s []int) int {
	sort.Ints(g)
	sort.Ints(s)
	children, cookies := 0, 0
	for children < len(g) && cookies < len(s) {
		if s[children] <= g[cookies] {
			children++
		}
		cookies++
	}
	return children
}
